src/Value/Color.php

```diff
@@ -66,14 +66,21 @@ public static function parse(ParserState $oParserState)
             $oParserState->consume('(');
 
             $bContainsVar = false;
-            $iLength = $oParserState->strlen($sColorMode);
+            if (strpos($sColorMode, 'rgb') !== false) {
```

Comment:
`rgb` has to be at the start, right?

```suggestion
            if (str_starts_with($sColorMode, 'rgb')) {
```

Reply:
Though I prefer being explicit:
```suggestion
            if ($sColorMode === 'rgb') {
```
(the `rgba` case will be handled in the `else` branch anyway).